### PR TITLE
start adding speakers to the site

### PR DESCRIPTION
**Pull request by evykassirer**
things to do before merging:

- [ ] pics for all speakers being added in this round
- [ ] ideally add the rest of what Anna has already
- [x] Keefer is sending me a new abstract so also waiting for that

But please check out the general layout and give feedback! thanks :)

### CODE REVIEW

File: _pages/speakers.md [hunk 1]
```diff
@@ -13,35 +13,96 @@ permalink: /speakers/
...
 
 ### Safia Abdalla
 
 ![](/assets/img/speakers/safia.jpg)
 
+<p class="bio">
 Safia is the founder of Zarf, a platform for independent writers who produce paid subscription content for their readers.
+</p>
 
 ## Invited lightning talk speakers
```
Comment (annalorimer) [inline _pages/speakers.md]: - Should all be capitalized 
- Maybe just "Invited Speakers"

File: _pages/speakers.md [hunk 2]
```diff
@@ -13,35 +13,96 @@ permalink: /speakers/
...
 
 ### Josh Bowman-Matthews
 
 ![](/assets/img/speakers/josh.jpg)
 
+<p class="bio">
 Josh Bowman-Matthews builds web browsers and sustainable communities for a living. He gets excited about demolishing barriers to participating in open source projects. He also sings in a barbershop quartet.
+</p>
+
+## Lightning talk speakers
```
Comment (annalorimer) [inline _pages/speakers.md]: - Should all be capitalized
- "Lightning talk speakers" sounds really awkward to me. How about just "speakers". Not sure what including the format of the conference adds to it

File: _pages/speakers.md [hunk 3]
```diff
@@ -13,35 +13,96 @@ permalink: /speakers/
...
+
+### Fatema Boxwala
+
+[Notworking with a Whale: A Tale of Dockers, Networks, and Problem Solving](/talks#notworking-with-a-whale-a-tale-of-dockers-networks-and-problem-solving)
+
+![](/assets/img/speakers/fatema.jpg)
```
Comment (annalorimer) [inline _pages/speakers.md]: I assume that the photos just haven't been added yet :)

File: _pages/speakers.md (hunk 2 above)
Comment (evykassirer) [reply _pages/speakers.md]: was trying to differentiate from keynotes, but I think it'd be clear enough as 'speakers' so I'll do that

keynotes / invited speakers / speakers

not too weird?

File: _pages/speakers.md (hunk 3 above)
Comment (evykassirer) [reply _pages/speakers.md]: oops I forgot to git add all

**Pull request merged**